composer.json

```diff
@@ -44,7 +45,22 @@
 		},
 		"bamarni-bin": {
 			"target-directory": "bin/composer"
-		}
+		},
+        "changelogger": {
+			"changelog": "./changelog.txt",
+        	"formatter": "WCAdmin",
+			"versioning": "wordpress",
+        	"changes-dir": "./changelogs",
```

Comment:
Some strange formatting here, I guess there might be some tabs in there.

Reply:
Intereting. It did not complain when I committed. I will fix it.